lib/modules/betteReddit.js

```diff
@@ -200,6 +205,9 @@ addModule('betteReddit', function(module, moduleID) {
 				default:
 					break;
 			}
+			if (module.options.truncateLongLinks.value) {
+				RESUtils.addCSS('overflow: hidden;text-overflow: ellipsis;white-space: nowrap;');
```

Comment:
@andytuba okay, will do. Is that all I have to do? Where would I put the boolean check?